app/invocation/invocation_execution_card.tsx

```diff
@@ -74,6 +76,12 @@ export default class ExecutionCardComponent extends React.Component<Props, State
         this.fetchUpdatedProgress();
       }
 
+      // To help with manual testing, support a special "openFirst" URL param
+      // which automatically navigates to the first execution that we fetched.
```

Comment:
Probably need to add a list of these somewhere (whether by an enum in code or just a list in a markdown file somewhere)--the way that they're placed right now means me and you will be the only people that know about them.